apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -333,7 +333,9 @@ suite('conversation.js >', function() {
     suite('rendering suggestions list', function() {
       var suggestionRenderer, unknownRenderer;
       var contact, unknown;
+      var clock;
       setup(function(done) {
+        clock = sinon.useFakeTimers(Date.now());
```

Comment:
Actually I need to remove all the `clock = sinon.useFakeTimers(Date.now());` as you suggested :)